script/deploy/zksync/UpdateCoreFacets.s.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { UpdateScriptBase, console } from "./utils/UpdateScriptBase.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { DiamondCutFacet, IDiamondCut } from "lifi/Facets/DiamondCutFacet.sol";
+import { DiamondLoupeFacet, IDiamondLoupe } from "lifi/Facets/DiamondLoupeFacet.sol";
+import { OwnershipFacet } from "lifi/Facets/OwnershipFacet.sol";
+import { WithdrawFacet } from "lifi/Facets/WithdrawFacet.sol";
+import { DexManagerFacet } from "lifi/Facets/DexManagerFacet.sol";
+import { AccessManagerFacet } from "lifi/Facets/AccessManagerFacet.sol";
+import { PeripheryRegistryFacet } from "lifi/Facets/PeripheryRegistryFacet.sol";
+import { StandardizedCallFacet } from "lifi/Facets/StandardizedCallFacet.sol";
+import { CalldataVerificationFacet } from "lifi/Facets/CalldataVerificationFacet.sol";
+
+contract DeployScript is UpdateScriptBase {
+    using stdJson for string;
+
+    function run()
+        public
+        returns (address[] memory facets, bytes memory cutData)
+    {
+        address diamondLoupe = json.readAddress(".DiamondLoupeFacet");
+        address ownership = json.readAddress(".OwnershipFacet");
+        address withdraw = json.readAddress(".WithdrawFacet");
+        address dexMgr = json.readAddress(".DexManagerFacet");
+        address accessMgr = json.readAddress(".AccessManagerFacet");
+        address peripheryRgs = json.readAddress(".PeripheryRegistryFacet");
+        address liFuelAddress = json.readAddress(".LIFuelFacet");
+        address genSwapAddress = json.readAddress(".GenericSwapFacet");
+        address genSwapV3Address = json.readAddress(".GenericSwapFacetV3");
+        address standCallAddress = json.readAddress(".StandardizedCallFacet");
+        address calldVerifAddress = json.readAddress(
+            ".CalldataVerificationFacet"
+        );
```

Comment:
same here - please take a look also on other places where`json.readAddress` is used.